src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,10 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
     public String getRandomColor() {
-        return null;
+        int colorNumber = new Random().nextInt(Color.values().length);
+        return Color.values()[colorNumber].toString();
```

Comment:
another point from the checklist
```suggestion
        return Color.values()[colorNumber].name();
```